Init. Know ur rights page developed

Splocko opened the pull request:
REDESIGNED MAIN PAGE
CREATED KNOW UR RIGHTS PAGE
<img width="1800" height="1169" alt="image" src="https://github.com/user-attachments/assets/df93df43-3705-4e6f-aee6-2a8c03f2f4dc" />
<img width="1800" height="1169" alt="image" src="https://github.com/user-attachments/assets/64053c16-ac84-45b8-98cb-56aacaf3edfc" />

END.

Commit:
Redesign main landing page with modern UI and vibrant styling

- Complete visual overhaul of main page with gradient backgrounds
- Added animated floating background elements with blur effects
- Implemented glassmorphism cards with hover animations
- Enhanced typography with gradient text effects
- Upgraded action buttons with 3D hover interactions
- Improved user section with modern styling
- Added colorful, engaging design that feels alive and professional

Commit:
Fix color palette and improve readability of Know Your Rights page

- Changed from dark theme to clean light theme for better readability
- Replaced harsh neon colors with soft, professional color scheme
- Improved contrast with dark text on white backgrounds
- Added subtle shadows and borders for better visual hierarchy
- Made color-coded sections more accessible and easier to scan
- Enhanced typography with proper contrast ratios for WCAG compliance
- Created professional, government-document-like appearance

Abdullah-Majid commented on src/app/know-your-rights/page.tsx:
```diff
@@ -0,0 +1,268 @@
+      
+      <main className="container mx-auto px-4 py-12">
+        <div className="grid grid-cols-1 lg:grid-cols-2 gap-8">
+          <div className="group bg-white rounded-xl p-8 border border-blue-200 hover:border-blue-300 transition-all duration-300 hover:shadow-lg hover:shadow-blue-100 shadow-sm">
```
I would make these into reusable components where we could inject the content in via props rather than a really long file of repeated code

Abdullah-Majid commented on src/app/page.tsx:
```diff
@@ -14,83 +14,167 @@ export default async function Home() {
+                  )}
+                  
+                  <div className="flex flex-col sm:flex-row gap-4">
+                    <Link
```
I would refactor this out into a component we could reuse as a header across pages

Commit:
Refactor into reusable components and improve code organization

- Created reusable PageHeader component for consistent headers across pages
- Added RightsCard component for standardized rights information display
- Created ArrestCard component for specialized arrest information layout
- Implemented UserStatusHeader and AuthButtons for modular authentication UI
- Centralized content in rights-data.ts for better maintainability
- Reduced know-your-rights page from 200+ lines to 47 lines
- Improved type safety with TypeScript interfaces
- Enhanced code reusability and DRY principles
